crates/holochain/tests/websocket/mod.rs

```diff
@@ -552,27 +553,47 @@ async fn conductor_admin_interface_ends_with_shutdown_inner() -> Result<()> {
 }
 
 #[tokio::test(flavor = "multi_thread")]
-async fn too_many_open() {
+#[cfg(feature = "slow_tests")]
+async fn connection_limit_is_respected() {
     holochain_trace::test_run().ok();
 
-    info!("creating config");
     let tmp_dir = TempDir::new().unwrap();
     let environment_path = tmp_dir.path().to_path_buf();
     let config = create_config(0, environment_path);
     let conductor_handle = Conductor::builder().config(config).build().await.unwrap();
     let port = admin_port(&conductor_handle).await;
-    info!("building conductor");
-    for _i in 0..1000 {
-        holochain_websocket::connect(
-            url2!("ws://127.0.0.1:{}", port),
-            Arc::new(WebsocketConfig {
-                default_request_timeout_s: 1,
-                ..Default::default()
-            }),
-        )
-        .await
-        .unwrap();
```

Comment:
This was my clue that this test wasn't covering the connection limit. It should have hit a limit at 400 and reported an error through this unwrap. There were two problems. The connection handles are being dropped which causes the client connections to immediately close (or nearly immediately) and means we never get anywhere near the limit. Also, unless you try to use the sender handle you don't actually know for sure you have a valid connection. Both these things are covered in the updated test.